lua/lvim/lsp/null-ls/services.lua

```diff
@@ -87,7 +87,21 @@ function M.register_sources(configs, method)
         compat_opts.args = nil
       end
 
-      local opts = vim.tbl_deep_extend("keep", { command = command }, compat_opts)
+      local check_exit_code
+      if method == null_ls.methods.FORMATTING then
+        check_exit_code = function(code, stderr)
+          local success = code <= 1
+
+          if not success then
+            Log:warn "Formatting failed!"
+            Log:debug(stderr)
```

Comment:
I would like to see `stderr` inside lvim, with a message that it is coming from null-ls/prettier, as it indicates where the mistake is. If it is not too noisy

Reply:
the message length is dependent on the formatter, and some may be long, also if the message is long, it will open up `messages`, that's why I put it in debug